tests/distributions/test_transforms.py

```diff
@@ -196,7 +196,8 @@ def test_conditional_radial(self):
         self._test_conditional(T.conditional_radial, inverse=False)
 
     def test_conditional_spline(self):
-        self._test_conditional(T.conditional_spline)
+        for order in ['linear', 'quadratic']:
+            self._test_conditional(partial(T.conditional_spline, order=order))
```

Comment:
nit: Does pytest notation work here?
```py
@pytest.mark.parametrize("order", ["linear", "quadratic"])
def test_conditional_spline(self, order):
    self._test_conditional(partial(T.conditional_spline, order=order))
```
That generally gives clearer test reports.

Reply:
It appears you can't do this on class methods: https://stackoverflow.com/questions/18182251/does-pytest-parametrized-test-work-with-unittest-class-based-tests?noredirect=1&lq=1

A workaround would involve requiring an extra dependency... It is better to keep as is?

Reply:
fine to keep as is.